src/services/refactors/convertExport.ts

```diff
@@ -152,8 +156,7 @@ namespace ts.refactor {
         }
     }
 
-    function changeImports(program: Program, { wasDefault, exportName, exportingModuleSymbol }: ExportInfo, changes: textChanges.ChangeTracker, cancellationToken: CancellationToken | undefined): void {
-        const checker = program.getTypeChecker();
+    function changeImports(program: Program, { wasDefault, exportName, exportingModuleSymbol }: ExportInfo, changes: textChanges.ChangeTracker, checker: TypeChecker, cancellationToken: CancellationToken | undefined): void {

```

Comment:
Why does changeImports need a checker parameter? No code changed.

Reply:
It was creating the checker locally before (in the line I removed after this line).  The ugly thing is that `changeExport` did require a checker argument, so I made them both symmetric in requiring the argument and using the same one for both (in `doChange`).

Reply:
Does this function work if the `checker` given to it didn't come from the same `program` ? Seems unlikely (?). Since the function is effectively free I'm not sure what's being gained here.

Reply:
They're both internal single-use functions, so no gain.  It just looked a little off that `.getTypeChecker` was used outside of one function and inside the other:

    function doChange(...) {
        changeExport(..., program.getTypeChecker());
        changeImports(program, ...);
    }
    function changeExport(..., checker) ...
    function changeImports(program, ...) {
        const checker = program.getTypeChecker();
        ...
    }

so I made it once outside,

    function doChange(...) {
        const checker = program.getTypeChecker();
        changeExport(..., checker);
        changeImports(..., checker);
    }
    function changeExport(..., checker) ...
    function changeImports(..., checker) ...

It's not important so if it's confusing I'll revert it.